new feature

Zony-Zhao opened the pull request:
1.Can read InputType class
2.Read Enum as String(sounds weird, but works if using flutter_graphql)

Zony-Zhao committed:
new feature:
1.Can read InputType class
2.Read Enum as String(sounds weird, but works if using flutter_graphql)

vikram25897 commented:
@Zony-Zhao  I haven't yet been able to test it. But I suggest you update the readme, version and changelog also.

Zony-Zhao commented:
It seems that still exist some issues when translating List of primitives. When I have List<String&gt; ,dart says List<Object&gt; is not subtype of List<String&gt;.
Without the '.map<String&gt;((o)=&gt;o.toString()).toList()' d which is I manually added. it throws error




------------------&nbsp;Original message&nbsp;------------------
From: "vikram25897"; 
Sendtime: Friday, Oct 16, 2020 7:09 PM
To: "vikram25897/graphql_to_dart"; 
Cc: "zonyzhao"<1668069501@qq.com>; "Mention"; 
Subject: Re: [vikram25897/graphql_to_dart] new feature (#3)





 
@Zony-Zhao  I haven't yet been able to test it. But I suggest you update the readme, version and changelog also.
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

vikram25897 commented:
@Zony-Zhao Okay I'll look at it and see what can be done.

Zony-Zhao commented:
![image](https://user-images.githubusercontent.com/50739849/96252964-5dc43800-0fe5-11eb-9873-6fcbb78c0ca1.png)

Zony-Zhao committed:
new feature:
1.Can read InputType class
2.Read Enum as String(sounds weird, but works if using flutter_graphql)

AlexQuidditch commented:
@vikram25897 @Zony-Zhao 
Any updates?
